apps/api/src/schema/enrollment-changes.ts

```diff
@@ -0,0 +1,105 @@
+import { z } from "@hono/zod-openapi";
+import { terms } from "@packages/db/schema";
+import { yearSchema } from "./lib";
+import { numCurrentlyEnrolledSchema, restrictionCodes, sectionStatusSchema } from "./websoc.ts";
+
+export const enrollmentChangesQuerySchema = z.object({
+  year: yearSchema,
+  quarter: z.enum(terms),
+  since: z
+    .string()
+    .datetime({ offset: false })
+    .transform((d) => new Date(d))
+    .openapi({
+      description: "The time which the `from` snapshot must describe; see route description.",
+    }),
```

Comment:
A bit more openapi docs here

```suggestion
    .string({message: "The 'since' query parameter is required."})
    .datetime({ offset: false, message: "The 'since' query must be a valid UTC datetime." })
    .transform((d) => new Date(d))
    .openapi({
      description: "The time which the `from` snapshot must describe; see route description.",
      example: "2025-02-19T00:00:00Z",
    }),
    ```